operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -58,6 +58,7 @@ kiali_defaults:
       custom_metrics_url: ""
       url: ""
     tracing:
+      enabled: true
```

Comment:
Just a question: do you really want `true` here and not the string `"true"`? - see my comment above